charts/minecraft-bedrock/templates/deployment.yaml

```diff
@@ -182,6 +182,9 @@ spec:
         {{- end }}
         securityContext:
           {{- toYaml .Values.securityContext | nindent 10 }}
+      {{- with .Values.extraContainers }}
```

Comment:
Let's call it `sidecarContainers` to be consistent with Java edition chart

https://github.com/itzg/minecraft-server-charts/blob/54f91138c1504d77245be86aced7ebb23e8eb7c9/charts/minecraft/values.yaml#L101